docs/source/augmentation.rst

```diff
@@ -16,6 +24,8 @@ The classes in this section perform various data augmentation operations
 .. autoclass:: CenterCrop
 .. autoclass:: RandomCrop
 .. autoclass:: RandomResizedCrop
+.. autoclass:: Normalize
+.. autoclass:: Denormalize
```

Comment:
it's fine by now, but we should start a discussion what goes in augmentation module and not.
To my view in augmentation we should only include stuff that implies generating random samples for a given op.
/cc @ducha-aiki  @mibaumgartner @justusschock

Reply:
In my opinion, I wish to build an API that can be used to replace Torchvision transform easily.

It will be the best if we can let a user update his code from ```from torchvision.transform import transform``` to ```import kornia.augmentation as transform```. The less coding, the better.

Reply:
Makes sense, another option is to expose every single module to the kornia top level
`import kornia as transforms`, so that we can internally organise as we want and not depending on third party decisions like torchvision or even opencv itself.

Reply:
In that sense, I think we shall back to the discussion of having a ```kornia.nn``` module to have all module-like classes grouped. In this way, all other parts could work individually but the user can choose ```import kornia.nn as transform``` as a shortcut to access all nn modules.

Reply:
We were working on making all transforms deterministic and adding randomness through parameter injection. This makes all transformations deterministic and thus makes testing easy :) With such an approach it is not really possible to differentiate between augmentation transformations (with randomness) and other transformations though. I like the idea of having something like `import kornia.augmentation as transform`or `import kornia.nn as transform` (while `kornia.augmentations` would be my favorite)